### PR TITLE
GOV-525 Stabilising G2pSandbox Helm Chart

### PR DESCRIPTION
**Pull request by somanath21**
(https://mifosforge.jira.com/jira/software/c/projects/GOV/issues/GOV-525?filter=allissues) Description

* Added Image name in ph-ee-engine chart.
* Disabled mock-payment-schema in ph-ee-engine
* Enabled mock payment schema in g2psandbox chart
* Updated Zeebe pod release 

 _(Ignore if these details are present on the associated JIRA ticket)_

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

**Review by fynmanoj — CHANGES_REQUESTED** [review]
update config maps as well

File: helm/ph-ee-engine/values.yaml [hunk 1]
```diff
@@ -15,12 +15,11 @@ configmap:
...
 zeebe:
   broker:
-    contactpoint: "zeebe-zeebe-gateway:26500"
-
+    contactpoint: "{{ .Release.Name }}-zeebe-gateway:26500"
 
 camunda-platform:
   global:
-    zeebe: "zeebe-zeebe"
+    zeebe: "{{ .Release.Name }}-zeebe"
```
Comment (fynmanoj) [inline helm/ph-ee-engine/values.yaml:22]: is this change needed?

File: helm/g2p-sandbox/values.yaml [hunk 2]
```diff
@@ -348,7 +349,7 @@ ph-ee-engine:
     imagePullPolicy: "Always"
     SPRING_PROFILES_ACTIVE: "bb"
     LOGGING_LEVEL_ROOT: "INFO"
-    zeebe_broker_contactpoint: "zeebe-zeebe-gateway:26500"
+    zeebe_broker_contactpoint: "g2p-sandbox-zeebe-gateway:26500"
```
Comment (fynmanoj) [inline helm/g2p-sandbox/values.yaml:352]: we can't hardcode this. instead, update this from deployment itself with if the condition
ie, if values are present then take from values, else by default take  {{ .Release.Name }}-zeebe-gateway:26500

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: helm/g2p-sandbox/values.yaml (hunk 2 above)
Comment (somanath21) [reply helm/g2p-sandbox/values.yaml]: Fixed in deployment.yaml

File: helm/ph-ee-engine/values.yaml (hunk 1 above)
Comment (somanath21) [reply helm/ph-ee-engine/values.yaml:22]: Fixed

**Review by fynmanoj — APPROVED** [review]
